docs/en/observability/profiling-troubleshooting.asciidoc

```diff
@@ -0,0 +1,235 @@
+
+The following is an example of a _healthy_ host-agent output:
+
+[source,logs]
+----
+time="..." level=info msg="Starting Prodfiler Host Agent v2.4.0 (revision develop-5cce978a, build timestamp 12345678910)"
+time="..." level=info msg="Interpreter tracers: perl,php,python,hotspot,ruby,v8"
+time="..." level=info msg="Automatically determining environment and machine ID ..."
+time="..." level=warning msg="Environment tester (gcp) failed: failed to get GCP metadata: Get \"http://169.254.169.254/computeMetadata/v1/instance/id\": dial tcp 169.254.169.254:80: i/o timeout"
+time="..." level=warning msg="Environment tester (azure) failed: failed to get azure metadata: Get \"http://169.254.169.254/metadata/instance/compute?api-version=2020-09-01&format=json\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)"
+time="..." level=warning msg="Environment tester (aws) failed: failed to get aws metadata: EC2MetadataRequestError: failed to get EC2 instance identity document\ncaused by: RequestError: send request failed\ncaused by: Get \"http://169.254.169.254/latest/dynamic/instance-identity/document\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)"
+time="..." level=info msg="Environment: hardware, machine ID: 0xdeadbeefdeadbeef"
+time="..." level=info msg="Assigned ProjectID: 5"
+time="..." level=info msg="Start CPU metrics"
+time="..." level=info msg="Start I/O metrics"
+time="..." level=info msg="Found tpbase offset: 9320 (via x86_fsbase_write_task)"
+time="..." level=info msg="Environment variable KUBERNETES_SERVICE_HOST not set"
+time="..." level=info msg="Supports eBPF map batch operations"
+time="..." level=info msg="eBPF tracer loaded"
+time="..." level=info msg="Attached tracer program"
+time="..." level=info msg="Attached sched monitor"
+----
+
+A host-agent deployment is working if the output of the following command is empty:
+
+[source,logs]
+----
+head host-agent.log -n 15 | grep "level=error"
+----
+
+If running this command outputs error-level logs, the following are possible causes:
+
+* The host-agent is running on an unsupported version of the Linux kernel, or its missing kernel features.
++
+If the host-agent is running on an unsupported kernel version, the following is logged:
++
+[source,logs]
+----
+Host Agent requires kernel version 4.15 or newer but got 3.10.0
+----
++
+If eBPF features are not available in the kernel, the host-agent fails to start, and one of the following is logged:
++
+[source,logs]
+----
+Failed to probe eBPF syscall
+----
++
+or
++
+[source,logs]
+----
+Failed to probe tracepoint
+----
+* The host-agent is not able to connect to {ecloud}. In this case, a similar message to the following is logged:
++
+[source,logs]
+----
+Failed to setup gRPC connection (retrying...): context deadline exceeded
+----
++
+Verify the `collection-agent` configuration value is set and is equal to what was printed by `elastic-profiling config`.

```

Comment:
@inge4pres as we now have a single click setup in Kibana, this is outdated, right? Does it make sense to fix this in a separate PR?

Reply:
Correct. I think we should amend it in this PR.
The instructions are printed directly from Kibana "Add Data", so we could say

```suggestion
Verify the `collection-agent` configuration value is set and is equal to what was printed in Kibana, when clicking to *Add Data*.
```